mockintosh/management.py

```diff
@@ -104,6 +119,33 @@ def check_restricted_fields(self, service, service_index):
                 raise Exception('%s field is restricted!' % field)
 
 
+class ManagementStatsHandler(tornado.web.RequestHandler):
+
+    def initialize(self, stats):
+        self.stats = stats
+
+    def get(self):
+        self.write(self.stats.json())
+
+    def delete(self):
+        self.stats.reset()
+        self.write('OK')
```

Comment:
If API is based on JSON, responses should not suddenly be arbitrary strings. In this specific case, I'd return `204 No content`. Or JSON response `true` if you want to report true/false status via JSON-standard way. This applies to all places where you return "OK" as string.

Reply:
Fixed with https://github.com/up9inc/mockintosh/commit/da16904e8a7868adce81c4263da10a1f054b62f8